### DISCUSSION TITLE
Issue with merging orientation maps

### DISCUSSION THREAD

**Discussion by mfuhr-mpg** [mtex-toolbox/mtex#1669] [Ask Anything] [answered]
Hi everybody,

I ran into an issue when trying to merge two orientation maps (map/ebsd 1 and map/ebsd 2):
![map1](https://user-images.githubusercontent.com/79193413/204504409-05a16a6b-1c91-4de1-890b-72908673c80a.jpg)
![map2](https://user-images.githubusercontent.com/79193413/204504444-7bdd019d-3ac0-4ab1-96c9-64857a0d4686.jpg)
The maps were measured with the same step size of 100 nm and an overlap of several µm. 

I use the following code to merge the datasets:
```
ebsd3 = shift(ebsd2,[113 -5.1]);
ebsd_merged = [ebsd1 ebsd3];
```

The result looks like this:
![merged](https://user-images.githubusercontent.com/79193413/204504899-2971724d-18d4-43a1-afb6-ad44bd824276.jpg)

If you take a closer look at the merged map, you see that the grains align nicely in the centre but are misaligned above and below.
Does anyone know why this is the case and how I can fix this issue? Do I for example have to resize the maps before merging so that all the grains line up? If so, how could I achieve that?

Thanks in advance!

Best
Max

**Comment by kilir** [comment, marked as answer]
Hi,
I can't reproduce this issue.Both have an identical output of ebsd.unitCell? Does shifting the other ebsd dataset in the opposite direction produce the same result? Does it work with other datasets and just not with this one?

Cheers,
Rüdiger

**Reply by mfuhr-mpg** [reply]
Hi Rüdiger,

the issue magically vanished when shifting the other dataset.
Thanks for the suggestion!

Best
Max
